### PR TITLE
remove undefined dns_servers output

### PR DESCRIPTION
**Pull request by ricoli**
As per the docs, the dns_servers attribute does not exist in the azurerm_virtual_network resource: https://www.terraform.io/docs/providers/azurerm/r/virtual_network.html#attributes-reference and as such needs to be removed as otherwise it will result in an error when using terraform >= 0.11.0

### CODE REVIEW

**Comment by AmirNaghibi** [comment]
Hi, could you explain the difference between argument and attribute? Because dns_server is in the argument reference but not in attribute reference (in the link you sent). I'm unclear about the difference.

**Comment by ricoli** [comment]
Hi Amir,
Attributes are what gets exported by the resource after it's been applied, which in turn can then become inputs (arguments) to other resources or modules.
TL;DR version: arguments = inputs, attributes = outputs
Hope this helps :)
